contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -8683,6 +8823,38 @@ handleGeospatialFunctionsInFunctionCall(TSqlParser::Function_callContext *ctx)
 	}
 }
 
+template<class T>
+void
+validateXMLFunctionArgs(T ctx)
+{
+	/* Only String Literal is allowed as agument for XML Functions */
+	if (ctx->xml_method_call() && ctx->xml_method_call()->expression_list())
```

Comment:
Redundant: According to currently defined rule `expression_list` will always be there in xml_method_call.

Reply:
refactored code